tools/offcputime.py

```diff
@@ -54,6 +54,11 @@ def positive_nonzero_int(val):
     help="kernel threads only (no user threads)")
```

Comment:
line 47 should delete "kernel". And I'd add a couple of lines after 44 so that -U and -K show up in the USAGE message, as they'll be commonly used I think.